tests/integration/bond_test.py

```diff
@@ -553,3 +556,57 @@ def test_set_miimon_100_on_existing_bond(bond99_with_2_slaves):
     bond_config[Bond.OPTIONS_SUBTREE] = {"miimon": 100}
     libnmstate.apply(state)
     assertlib.assert_state_match(state)
+
+
+@pytest.fixture
+def eth1_eth2_with_no_profile():
```

Comment:
If you do not provide the fixtures `ethX_up`, there should be no profile. Isn't this the case?
The only place I think you need a fixture is for the teardown, where you probably need to call `ifacelib.ifaces_init("eth1", "eth2")`.

Reply:
Do we have something that guarantees that all profiles are removed before a test? Otherwise there might be a leftover profile if a test does not properly clean up.

Reply:
The fixture `eth1_up` and `eth2_up` will remove the profile at clean up. So we can assume this test case started with
empty eth1 and eth2.